data/aliases.js

```diff
@@ -956,56 +956,58 @@ let BattleAliases = {
 	"gyaradosu": "Gyarados",
 	"rapurasu": "Lapras",
 	"metamon": "Ditto",
-	"bui": "Eevee",
+	"iibui": "Eevee",
 	"shawaazu": "Vaporeon",
 	"sandaasu": "Jolteon",
 	"buusutaa": "Flareon",
-	"porigon": "Porygon",
+	"porigon": "Porygon-Z",
```

Comment:
This is likely a mistake

Reply:
Done. I don't have anything for Porygon2 or Porygon-Z though. Should they just be `porigonz` and `porigon2` do you think?

Reply:
yeah

Reply:
Done.